inst/validation/specs.R

```diff
@@ -0,0 +1,28 @@
+# Use a list to declare the specs
+# nolint start
```

Comment:
Should we restrict to line_length_linter?

Reply:
Yes, we can also do it like this :)